packages/Parallel/Application/ParallelFileProcessor.php

```diff
@@ -42,8 +42,20 @@ final class ParallelFileProcessor
      */
     private const SYSTEM_ERROR_LIMIT = 50;
 
+    private int $maxProcessIterations = 10;
```

Comment:
constant can be used instead as seems never changed later

Reply:
yes, for as it is yes. Idea was that this is something which is controlled over a setting in the future, so having it directly as a variable would reduce needed changes for the future.

But I can of course change it to a constant if requested